### PR TITLE
Add parentheses for all method calls

### PR DESCRIPTION
**Pull request by KrauseFx**
Using rubocop

This is amazing, have been waiting for this for such a long time, thanks @milch

### CODE REVIEW

**Review by milch — APPROVED** [review]
400 files changed, nice 😄

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by KrauseFx** [comment]
This is gonne cause conflicts in many other PRs, but we gotta merge at one point, so why not now? :)

File: fastlane_core/lib/fastlane_core/PTY.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@ def self.spawn(*cmd, &block)
         require "systemu"
         stdout = ''
         stderr = ''
-        _status = systemu cmd, 'stdout' => stdout, 'stderr' => stderr
+        _status = systemu(cmd, 'stdout' => stdout, 'stderr' => stderr)
```
Comment (janpio) [inline fastlane_core/lib/fastlane_core/PTY.rb]: You should rebase onto master - this is not actually in there any more ;)

**Comment by KrauseFx** [comment]
Yep, gone, thanks @janpio

**Comment by revolter** [comment]
Finally, consistency and cleanliness ❤️  Thank you!

**Pull request merged**